### ISSUE TITLE
COCO performance after finetuning

### ISSUE THREAD

**Issue by HaisongDing** [FoundationVision/GLEE#14]
Have you tried to finetune the stage 2 model on COCO dataset and see the limit of the COCO performance?

**Comment by wjf5203** [comment]
Yes, we have observed that continuing to finetune GLEE-Lite-Joint on COCO does not improve performance. We believe this highlights the advantage of GLEE—enhancing the model's detection abilities by training with a larger amount of data. However, the performance of GLEE-Pro on COCO has not yet met expectations, and we are currently exploring the reasons for this.

**Comment by ayushP-uwc** [comment]
> Have you tried to finetune the stage 2 model on COCO dataset and see the limit of the COCO performance?

Hey, could you please tell me how you fine-tuned it on the COCO dataset?
